streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -63,20 +64,24 @@ FhirConverter createFhirConverter(CamelContext camelContext) throws IOException,
 		ParquetUtil parquetUtil = new ParquetUtil(params.fileParquetPath, params.secondsToFlushParquetFiles,
 		        params.rowGroupSizeForParquetFiles);
 		camelContext.addService(new ParquetService(parquetUtil), true);
-		return new FhirConverter(openmrsUtil, fhirStoreUtil, parquetUtil, params.fhirDebeziumEventConfigPath);
+		return new FhirConverter(openmrsUtil, fhirStoreUtil, parquetUtil, params.fhirDebeziumConfigPath);
 	}
 	
-	private String getDebeziumConfig() {
-		return "debezium-mysql:" + params.databaseHostName + "?" + "databaseHostname=" + params.databaseHostName
-		        + "&databaseServerId=" + params.databaseServerId + "&databasePort=" + params.databasePort.intValue()
-		        + "&databaseUser=" + params.databaseUser + "&databasePassword=" + params.databasePassword
-				//+ "&name={{database.dbname}}"
-		        + "&databaseServerName=" + params.databaseName + "&databaseWhitelist=" + params.databaseSchema
-		        + "&offsetStorage=org.apache.kafka.connect.storage.FileOffsetBackingStore" + "&offsetStorageFileName="
-		        + params.databaseOffsetStorage + "&databaseHistoryFileFilename=" + params.databaseHistory + "&snapshotMode="
-		        + params.snapshotMode
-		//+ "&tableWhitelist={{database.schema}}.encounter,{{database.schema}}.obs"
-		;
+	private String getDebeziumConfig() throws IOException {
+		GeneralConfiguration gc = new GeneralConfiguration();
+		GeneralConfiguration generalConfiguration = gc.getFhirDebeziumConfigPath(params.fhirDebeziumConfigPath);
+		return "debezium-mysql:" + generalConfiguration.getDbzConfigs().get("databaseHostName") + "?" + "databaseHostname="
+		        + generalConfiguration.getDbzConfigs().get("databaseName") + "&databaseServerId="
+		        + generalConfiguration.getDbzConfigs().get("databaseServerId") + "&databasePort="
```

Comment:
@bashir2 thanks for the review 
yes we need them forexample when you remove the databaseServerId you get this[ error ](https://pastebin.com/u189crc8) upon running the pipeline

Reply:
I am talking about `DebeziumArgs` class. Is `DebeziumArgs.databaseServerId` used anywhere in the code?

Reply:
@bashir2 removing `DebeziumArgs.databaseServerId` from the `DebeziumArgs `class fails the pipeline with `com.beust.jcommander.ParameterException: Was passed main parameter '--databaseServerId=77' but no main parameter was defined in your arg class`
its used in the debezium configs code

Reply:
Did you checked why is that? Where is this coming from? unit-tests? Was this coming from running `docker-compose.yaml`, because I can see that there is a `--databaseServerId=77` there., which should not be needed after your change.

Reply:
@bashir2 i did check and its because the `--databaseServerId=223344` argument represents the server-id set in the `mysql.cnf` file when defining replication and binary logging options and variables.
A sample example [here](https://debezium.io/documentation/reference/0.10/connectors/mysql.html#example-configuration) still needs the data base server-id.

Reply:
I am not saying we should not set `databaseServerId` when setting up Debezium. What I am saying is that you are reading that value from the JSON config file, therefore you should not need the `--databaseServerId` *command line argument*. Are you passing `--databaseServerId` when you are running the pipeline?

Is your TESTED field in PR description up-to-date? There, you have `--databaseServerId=223344` (you shouldn't), what happens if you remove this? Please update the TESTED field with the latest version of what you run and if it fails put the full log somewhere for me to check.

Reply:
@bashir2 works fine,i have updated the read-me file and removed other un-needed commandline args

Reply:
Okay, good, but you have not  addressed the original comment in this thread: Please remove all unused command line arguments from `DebeziumArgs`.

Reply:
@bashir2 sure,,its done